package.json

```diff
@@ -32,6 +32,9 @@
   },
   "scripts": {
     "start": "appc run",
+    "check-deps": "retire || true",
+    "clean": "rimraf ./tmp",
+    "checks": "npm run check-deps && npm run check-security",
     "test": "npm run lint && npm run test:unit",
```

Comment:
**checks** must be run before **lint** in the **test** script so they are executed in Jenkins builds. Moreover it could be useful to add **test-dev** script contains **lint** plus **test:unit** without the **checks**